src/App.js

```diff
@@ -1,16 +1,25 @@
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
 import { Routes, Route } from 'react-router-dom';
 import NavBar from './components/NavBar/NavBar';
-import Missions from './pages/Missions';
+import MissionsPage from './pages/Missions';
 import Profile from './pages/Profile';
 import Rockets from './pages/Rockets';
+import { getMissionsList } from './redux/missions/missions';
 
 function App() {
+  // dispatch the actions
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getMissionsList());
+  }, []);
```

Comment:
Excellent use of the useEffect hook 👍🏻 I am going to use this part for fetching the rocket's data.
however, in the case of the missions, the requirement specifies that the fetching must be done when the user navigates to the mission section. 
#Fetch data from the Missions endpoint when a user navigates to the Missions section.
Please consider this change for future code changes, as I have already merged this branch into my code.

Reply:
Thank you so much. I was in two minds about that. Anyway, as the next task is to continue working on that, I will update it.